src/qt/walletcontroller.cpp

```diff
@@ -431,7 +431,7 @@ void RestoreWalletActivity::finish()
         QMessageBox::warning(m_parent_widget, tr("Restore wallet warning"), QString::fromStdString(Join(m_warning_message, Untranslated("\n")).translated));
     } else {
         //: Title of message box which is displayed when the wallet is successfully restored.
-        QMessageBox::information(m_parent_widget, tr("Restore wallet message"), QString::fromStdString(Untranslated("Wallet restored successfully \n").translated));
+        QMessageBox::information(m_parent_widget, tr("Restore wallet message"), QString::fromStdString(Untranslated("Wallet restored successfully \n").translate()));
```

Comment:
re: https://github.com/bitcoin/bitcoin/pull/31061#discussion_r1793707636

Would be reasonable to change this but no more change is actually required here with my suggested branch.

Reply:
re: https://github.com/bitcoin/bitcoin/pull/31061#discussion_r1793707636

Simplified this further in 6fb39655c5cd2bebad902d271d1edae64b817d2b from #31072. It might just be a bug that this string is not translated, and there also seems to be trailing whitespace in the string. Current commit just simplifies code and keeps existing behavior though.